ddln_imaginator.proto

```diff
@@ -21,6 +21,64 @@ enum ImageFormat {
     GRAY = 4;
 }
 
+message Environment {
+    message Clouds {
+        enum Type {
+            NONE = 0;
+            ALTOCUMULUS = 1;
+            CLOUDS_BASE = 2;
+            CUMULUS = 3;
+            NIMBOSTRATUS = 4;
+            STRATOCUMULUS = 5;
+            STRATUS = 6;
+        }
+        optional Type type = 1;
+        optional float altitude_m = 2;
+        optional Point3d velocity = 3; // Only use the directional x,y,z values to set the wind in the Engine.
+    }
+    optional Clouds clouds = 1;
```

Comment:
usually it is possible to have multiple cloud levels, does it make sense to make this field repeatable?

Reply:
High altitude clouds could be added as a sky texture and might be a future addition. Let's keep it simple for now.

Reply:
It is important to have interface that we can extend without breaking code or wire compatibility.
So lets make it repeatable right away, even if we don't support multiple layers in v1.

Reply:
Alex also asked if it's possible to make multiple layers. So, I'll make it 'repeated' instead of 'optional'.

Reply:
Please contact Alex before making changes. Maybe two layers might be enough and this scenario can be covered by high altitude clouds.

Reply:
Ok, I'll ask him.

Reply:
Max three is demanded and I'll make it repeatable.